plugins/GSdx/GSRendererHW.cpp

```diff
@@ -201,22 +201,23 @@ void GSRendererHW::InvalidateLocalMem(const GIFRegBITBLTBUF& BITBLTBUF, const GS
 	m_tc->InvalidateLocalMem(m_mem.GetOffset(BITBLTBUF.SBP, BITBLTBUF.SBW, BITBLTBUF.SPSM), r);
 }
 
-int GSRendererHW::Interpolate_UV(float alpha, int t0, int t1)
+uint16 GSRendererHW::Interpolate_UV(float alpha, int t0, int t1)
 {
 	float t = (1.0f - alpha) * t0 + alpha * t1;
-	return (int)t & ~0xF; // cheap rounding
+	return (uint16)t & ~0xF; // cheap rounding
 }
 
+//X1 not even used?? Should this be similar to alpha1 with different rounding?
 float GSRendererHW::alpha0(int L, int X0, int X1)
 {
-	float x = (X0 + 15) & ~0xF; // Round up
-	return (x - X0) / (float)L;
+	int x = (X0 + 15) & ~0xF; // Round up
+	return float(x - X0) / (float)L;
 }
```

Comment:
Keep it. It could still be useful if someone try to tune the algo.
However be aware of the uint16 cast. It could overflow. It is surely safer to cast it only after the various operation